packages/stark-core/testing.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./src/http/testing/index";
+export * from "./src/logging/testing/index";
+export * from "./src/routing/testing/index";
+export * from "./src/session/testing/index";
```

Comment:
Question.... isn't it better to have a separate re-export for each of those inside their own folder? So the developer can import just the mock he needs... for example, in Angular, when you want to import the HttpClient mock you do something like:

`import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing'`